dags/utils/acoustic/acoustic_client.py

```diff
@@ -188,90 +186,3 @@ def generate_report(
         logging.info(f"{report_type} generation complete. Report location: {report_loc}. Time taken: {datetime.now() - start}")
 
         return
-
```

Comment:
Are you sure this file isn't needed anywhere else?

Reply:
hey, no it's not. This is the only place.